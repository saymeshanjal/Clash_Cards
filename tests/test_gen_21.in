90
6 106 1
10 99 1
8 152 1
14 195 3
12 140 1
12 313 4
8 297 1
10 196 5
10 320 4
6 305 3
12 371 4
4 199 4
6 139 3
14 73 1
14 222 3
4 299 1
16 286 5
14 282 1
12 334 4
8 192 4
4 309 5
6 326 2
14 200 1
6 95 1
6 309 3
16 390 4
10 165 5
8 268 2
12 137 3
2 244 3
10 213 2
10 55 4
8 365 4
12 48 4
2 2 5
4 369 1
12 214 3
16 185 3
8 174 5
12 359 1
2 243 2
4 199 2
2 74 3
16 263 3
14 135 2
4 216 4
8 5 5
8 140 5
14 103 1
8 229 5
8 261 1
4 110 5
16 54 2
6 390 5
12 268 3
6 93 4
4 161 2
14 188 4
10 42 3
4 261 2
10 193 3
8 50 1
6 289 5
8 75 2
8 112 5
8 51 1
10 212 4
6 315 2
10 191 1
14 368 5
12 357 3
6 352 3
12 202 5
14 237 5
14 17 5
4 380 1
6 344 3
10 314 3
6 335 5
8 157 5
4 363 2
2 61 5
4 66 4
10 32 4
8 398 1
6 136 3
10 138 3
6 119 4
10 169 4
12 28 4
