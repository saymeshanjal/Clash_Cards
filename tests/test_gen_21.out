10 10 0
126 126 0
35 35 0
1652 1780 0
462 462 0
208 441 275
35 35 0
45 207 0
44 133 75
6 14 0
208 441 275
0 5 1
6 14 0
1716 1716 0
1652 1780 0
3 3 0
4248 8622 0
1716 1716 0
208 441 275
8 42 20
0 6 0
5 6 9
1716 1716 0
10 10 0
6 14 0
3809 5406 3655
45 207 0
20 21 29
430 494 0
0 2 0
76 77 99
44 133 75
8 42 20
195 441 288
0 2 0
3 3 0
430 494 0
6307 6563 0
8 62 0
462 462 0
0 1 1
1 2 3
0 2 0
6307 6563 0
1078 1079 1275
0 5 1
0 70 0
8 62 0
1716 1716 0
8 62 0
35 35 0
0 6 0
1424 1928 9518
1 19 0
430 494 0
1 14 5
1 2 3
910 1520 1002
110 142 0
1 2 3
110 142 0
35 35 0
1 19 0
20 21 29
8 62 0
35 35 0
44 133 75
5 6 9
126 126 0
987 2445 0
430 494 0
6 14 0
219 705 0
987 2445 0
42 3390 0
3 3 0
6 14 0
110 142 0
1 19 0
8 62 0
1 2 3
0 2 0
0 5 1
20 137 95
35 35 0
6 14 0
110 142 0
1 14 5
44 133 75
20 722 182
